lib/Doctrine/Inflector/Rules/Italian/Inflectible.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Inflector\Rules\Italian;
+
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Substitution;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Word;
+
+class Inflectible
+{
+    /**
+     * @return Transformation[]
+     */
+    public static function getSingular() : iterable
+    {
+        yield new Transformation(new Pattern('e$'), 'a');
+        yield new Transformation(new Pattern('i$'), 'e');
+        yield new Transformation(new Pattern('i$'), 'o');
```

Comment:
This transformation will never happen given the previous one has exactly same catch pattern